composer.json

```diff
@@ -39,7 +39,9 @@
     "require": {
         "php": "^5.5.0|^7.0",
         "payum/iso4217": "~1.0",
-        "guzzlehttp/guzzle": "~6.0",
+        "php-http/client-implementation": "^1.0",
+        "php-http/discovery": "^0.8",
+        "guzzlehttp/psr7": "^1.1",
```

Comment:
@Nyholm As soon as composer fails on update when there is not an implementation for `php-http/client-implementation` we are safe and can get rid of guzzle as default package. Is this how it expected to work?